guides/0115-filter.md

```diff
@@ -0,0 +1,51 @@
+# filter
+
+* type: producer_consumer
+* input: any kind of message
+* output: same message type as input
+
+This is a producer and consumer block that filters messages that get rejected by the filter script.
+
+This block supports [Lua 5.2](https://www.lua.org/manual/5.2/) scripts (
+[luerl scripting engine](https://github.com/rvirding/luerl) is used under the hood). The incoming
+message will be provided to the script as `message`.
+
+# Properties
+
+* `script`: The LUA script used to filter incomming messages. (required, string)
+
+## `script`
+
+The LUA script that will determine if the message has to be discarded or allowed through.
```

Comment:
Lua, not LUA

Reply:
This explanation is not really clear, it doesn't explain what this script does and how it behaves.

Reply:
More important we must state that the Lua script must evaluate to a boolean value, that can be used to decide whether the message is passed through or discarded.

Reply:
I'll add some more detailed description exaplaining which retuned values allows the message to pass or to get discarded